Scripts/BuildPhases/GenerateCredentials.sh

```diff
@@ -47,12 +47,8 @@ function ensure_is_in_input_files_list() {
   fi
 }
 
-PRODUCTION_SECRETS_FILE="${SECRETS_ROOT}/WordPress-Secrets.swift"
-ensure_is_in_input_files_list $PRODUCTION_SECRETS_FILE
-INTERNAL_SECRETS_FILE="${SECRETS_ROOT}/WordPress-Secrets-Internal.swift"
```

Comment:
- `WordPress-Secrets-Internal` was unused.
- `WordPress-Secrets-Alpha` is only specified for the WordPress and is identical to the production. The only difference is `sentryDSN` which appears to be is incorrectly specified – probably worth checking.